data_structures/src/superblock.rs

```diff
@@ -142,7 +156,10 @@ impl SuperBlockState {
                 }
                 None => AddSuperBlockVote::MaybeValid,
             }
-        }
+        };
+        log::debug!("Add vote: {:?}", r);
```

Comment:
This log is currently useful, but when we merge everything, it wont be necessary. So I would put a TODO comment to remember delete this log